lib/db.js

```diff
@@ -261,6 +262,34 @@ module.exports = (
     log.trace({ op: 'DB.sessions', uid: uid })
     const promises = [
       this.pool.get('/account/' + uid + '/sessions')
+        .then(sessionTokens => {
```

Comment:
Found this by accident, if a session has a device and then the user signs out of that session, the session gets deleted from MySQL but not Redis. Does [deleteSession](https://github.com/mozilla/fxa-auth-server/blob/680f25f5a9f3454b0db585528e44872074187ffe/lib/db.js#L775) need the pack/unpack redis logic?

Reply:
Oh, great catch! Definitely a bug, I've raised #2260.